__test__/actions/signinAction.test.js

```diff
@@ -0,0 +1,63 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import nock from 'nock';
+import * as actions from '../../app/client/actions/signinAction';
+import * as types from '../../app/client/actions/types';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+
+describe('Sign in action creators tests', () => {
+  describe('Sync action creators', () => {
```

Comment:
Nice job with your tests , but your assertion descriptions should correlate, '_Sign in action creator tests **Sync action creators** should create an action to set current user_', 
So you could make your assertion messages be of something like this:
 'Sign in action creator that sync action creators should create an action to set current user'